Admin/AbstractAdmin.php

```diff
@@ -1626,6 +1626,19 @@ public function hasParentFieldDescription()
      */
     public function setSubject($subject)
     {
+        if (!is_a($subject, $this->class, true)) {
+            $message = <<<EOT
```

Comment:
Apparently, StyleCI wants `<<<'EOT'` here, because there are no variables to interpret